src/sources/PendoClient.js

```diff
@@ -44,7 +44,20 @@ const Pendo = {
     // R.map((key) => R.partial(Pendo[key], token), R.reject(R.equals('initialize'),R.keys(Pendo)))
   },
 
+  setUrl (token) {
```

Comment:
You already mentioned not handling the `.us` correct but it also looks like you're not updating the `token` to drop the `.<region>` suffix after updating the URL.